docs/AuthProviderWriting.md

```diff
@@ -402,6 +404,10 @@ React-admin doesn't use permissions by default, but it provides [the `usePermiss
 
 [The Role-Based Access Control (RBAC) module](./AuthRBAC.md) allows fined-grained permissions in react-admin apps, and specifies a custom return format for `authProvider.getPermissions()`. Check [the RBAC documentation](./AuthRBAC.md#authprovider-methods) for more information.
 
+### `canAccess`
+
+This method should return a boolean indicating whether users can perform the provided action on the provided resource (in the [Role-Based Access Control (RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control) sense).
+
```

Comment:
This will need a rewrite to list the calls made by react-admin in various places. Also, we shouldn't mention only RBAC as this syntax also allows ABAC, ACL, and more. 

Also, it's not clear by reading the doc what the authProvider should return when the user doesn't have the rights for a resource (a rejected Promise or a resolved Promise to false?)